scripts/exrrfs_make_sfc_climo.sh

```diff
@@ -105,12 +105,12 @@ cd $workdir
 cat << EOF > ./fort.41
 &config
 input_facsf_file="${SFC_CLIMO_INPUT_DIR}/facsf.1.0.nc"
-input_substrate_temperature_file="${SFC_CLIMO_INPUT_DIR}/substrate_temperature.2.6x1.5.nc"
+input_substrate_temperature_file="${SFC_CLIMO_INPUT_DIR}/substrate_temperature.gfs.0.5.nc"
 input_maximum_snow_albedo_file="${SFC_CLIMO_INPUT_DIR}/maximum_snow_albedo.0.05.nc"
 input_snowfree_albedo_file="${SFC_CLIMO_INPUT_DIR}/snowfree_albedo.4comp.0.05.nc"
 input_slope_type_file="${SFC_CLIMO_INPUT_DIR}/slope_type.1.0.nc"
-input_soil_type_file="${SFC_CLIMO_INPUT_DIR}/soil_type.statsgo.0.05.nc"
-input_vegetation_type_file="${SFC_CLIMO_INPUT_DIR}/vegetation_type.igbp.0.05.nc"
```

Comment:
@BenjaminBlake-NOAA, I have a question for you. Doesn't the above file change (3 files) cause any issues to others than FW?

Reply:
Yes.  Would it make sense to create a variable name for these 3 files and set default values for them as the current files?

Reply:
Yes, it is. Or you can add an if-statement for the specific fire weather domain or a flag for fire-weather.

Reply:
I added an if statement to use these specific files when running with the fire weather grid.  Let me know if that works.